src/lib/ui/noise-sensor/NoiseThresholdsList.tsx

```diff
@@ -0,0 +1,122 @@
+
+import { formatTime, formatTimeZone } from 'lib/dates.js'
+import { ArrowRightIcon } from 'lib/icons/ArrowRight.js'
+import { useNoiseThresholds } from 'lib/seam/noise-sensors/use-noise-thresholds.js'
+import { DetailRow } from 'lib/ui/layout/DetailRow.js'
+import { DetailSection } from 'lib/ui/layout/DetailSection.js'
+import { DetailSectionGroup } from 'lib/ui/layout/DetailSectionGroup.js'
+
+interface NoiseThresholdsListProps {
+  device: NoiseSensorDevice
+}
+
+export function NoiseThresholdsList({
+  device,
+}: NoiseThresholdsListProps): JSX.Element {
+  const { noiseThresholds, isInitialLoading } = useNoiseThresholds({
+    device_id: device.device_id,
+  })
+
+  return (
+    <>
+      <DetailSectionGroup>
+        <div className='seam-detail-section-wrap'>
+          <DetailSection
+            label={t.noiseThresholds}
+            tooltipContent={
+              device.device_type === 'minut_sensor' ? (
+                <div className='seam-detail-section-tooltip-inner-content'>
+                  <span className='seam-tooltip-content'>
+                    {t.minutTooltipFirst}
+                  </span>
+                  <span className='seam-tooltip-content'>
+                    {t.minutTooltipSecond}
+                  </span>
+                </div>
+              ) : (
+                t.tooltip
+              )
+            }
+          >
+            {!isInitialLoading && (noiseThresholds?.length ?? 0) > 0 ? (
+              noiseThresholds?.map((noiseThreshold) => (
+                <DetailRow
+                  key={noiseThreshold.noise_threshold_id}
+                  label={
+                    <div className='seam-detail-row-label-column'>
+                      {noiseThreshold.name !== '' && (
+                        <span className='seam-detail-row-label'>
+                          {noiseThreshold.name}
+                        </span>
+                      )}
+                      <div className='seam-detail-row-label-block'>
+                        <span className='seam-row-sublabel seam-row-sublabel-text-default'>
+                          {formatTime(noiseThreshold.starts_daily_at)}
+                        </span>
+                        <ArrowRightIcon />
+                        <span className='seam-row-sublabel seam-row-sublabel-text-default'>
+                          {formatTime(noiseThreshold.ends_daily_at)}
+                        </span>
+                      </div>
+                    </div>
+                  }
+                >
+                  <p>
+                    {noiseThreshold.noise_threshold_decibels} {t.decibel}
+                  </p>
+                </DetailRow>
+              ))
+            ) : (
+              <DetailRow
+                label={
+                  <span className='seam-detail-row-empty-label'>{t.none}</span>
+                }
+              />
+            )}
+          </DetailSection>
```

Comment:
This ternay structure is hard to parse. I was hoping that by moving it to a component, we can have a more linear, easy to follow style, e.g.


```
  const { noiseThresholds, isInitialLoading } = useNoiseThresholds({
    device_id: device.device_id,
  })
if (isInitialLoading) return null
if (noiseThresholds == null) return null
return noiseThresholds.map(noiseThreshold => <NoiseThreshold key={noiseThreshold.noise_threshold_id} noiseThreshold={noiseThreshold} />